langs.toml

```diff
@@ -347,6 +347,20 @@ end
 size    = '33.6 MiB'
 version = 'GNU CLISP 2.49.92'
 website = 'https://clisp.sourceforge.io'
+example = '''
+; Printing
+(write-line "Hello World")
```

Comment:
This needs to be `Hello, World!` (with a comma and a trailing bang) to pass `t/examples.t` (e2e doesn't currently work on GH due to an unrelated issue)

Reply:
You should also now remove a TODO in `t/examples.t`:
```diff
--- a/t/examples.t
+++ b/t/examples.t
@@ -3,9 +3,6 @@ use t;
 for slurp("langs.toml").&from-toml.map({
     .key.lc.trans( qw[# ><>] => qw[-sharp fish] ) => .value<example>;
 }).sort -> (:key($lang), :value($code)) {
-    # TODO Remove this to ensure we have examples for every lang.
-    next unless $code;
-
     # <built-in>: internal compiler error: Illegal instruction
     todo 'intermittent error' if $lang eq 'fortran';
```